<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetGameWindow</class>
 <widget class="QMainWindow" name="NetGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>局域网对战</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/image/gobang.ico</normaloff>:/image/image/gobang.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="messageBrowser">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(178, 113, 60);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="messageTextEdit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>600</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="userTableWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(178, 113, 60);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>主机名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP地址</string>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="readyBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>640</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 楷体 Std R</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>准备</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="giveupBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>640</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 楷体 Std R</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>认输</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe 楷体 Std R</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>悔棋</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>640</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 楷体 Std R</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>107</y>
       <width>134</width>
       <height>119</height>
      </rect>
     </property>
     <property name="title">
      <string>新游戏</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image/image/new.png</normaloff>:/image/image/new.png</iconset>
     </property>
     <addaction name="actionPeople"/>
     <addaction name="actionComputer"/>
     <addaction name="actionNet"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="actionHome"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="actionBack"/>
    <addaction name="actionScreenshot"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/quit.png</normaloff>:/image/image/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/back.png</normaloff>:/image/image/back.png</iconset>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </action>
  <action name="actionPeople">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/person.png</normaloff>:/image/image/person.png</iconset>
   </property>
   <property name="text">
    <string>人人对战</string>
   </property>
  </action>
  <action name="actionComputer">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/computer.png</normaloff>:/image/image/computer.png</iconset>
   </property>
   <property name="text">
    <string>人机对战</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/screenshot.png</normaloff>:/image/image/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>保存棋局</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/home.png</normaloff>:/image/image/home.png</iconset>
   </property>
   <property name="text">
    <string>回到菜单</string>
   </property>
  </action>
  <action name="actionNet">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/net.png</normaloff>:/image/image/net.png</iconset>
   </property>
   <property name="text">
    <string>局域网对战</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
